openmrs/apps/reports/sql/offlineOPDVisitCount.sql

```diff
@@ -0,0 +1,7 @@
+SELECT u.name AS User, c.name AS Location, COUNT(DISTINCT e.individual_id) AS "Offline OPD Visit Count"
```

Comment:
Count of distinct individual id will give only the number of patients that had an encounter in the date range. This should be grouped by date also and then sum should be taken